Web/HTML/core-html-elements/update-hyperlinks.md

```diff
@@ -52,21 +125,28 @@ How is a placeholder link created, where the link works, but it doesn't open a n
 
 `<a href="???">Empty href</a>`
 
-* #
-* ""
-* *
-* _empty
-* _self
-* Leave empty
+* `#`
+* `""`
+* `*`
+* `_empty`
+* `_self`
+* `Leave empty`
 
 ---
 ## Quiz
 
+### How much do you know about HTML hyperlinks?
```

Comment:
Only one of these questions can be in the Quiz section, so pull out some of these into the Revision question.